deps/db/src/logseq/db.cljs

```diff
@@ -4,23 +4,99 @@
             [logseq.db.frontend.schema :as db-schema]
             [datascript.core :as d]
             [cljs-time.core :as t]
-            [cljs-time.coerce :as tc]))
+            [cljs-time.coerce :as tc]
+            [clojure.string :as string]
+            [logseq.common.util :as common-util]
+            [logseq.common.config :as common-config]
+            [logseq.db.frontend.content :as db-content]
+            [clojure.set :as set]
+            [logseq.db.frontend.rules :as rules]
+            [logseq.db.frontend.entity-plus]
+            [promesa.core :as p]))
+
+;; Use it as an input argument for datalog queries
+(def block-attrs
```

Comment:
Some fns in `frontend.db.model` has been moved here so that it can be used in the db dep and others, we might move more fns here in the future.